src/code/providers/google-drive-provider.coffee

```diff
@@ -195,8 +211,10 @@ class GoogleDriveProvider extends ProviderInterface
   _loadGAPI: ->
     if not window._LoadingGAPI
       window._LoadingGAPI = true
-      window._GAPIOnLoad = ->
-        @window._LoadedGAPI = true
+      window._GAPIOnLoad = =>
+        window._LoadedGAPI = true
+        # preload clients to avoid user delay later
+        @_loadedGAPI ->
```

Comment:
The intent was to call the `_loadedGAPI()` method with an empty callback. No instance variables were set on the GoogleDriveProvider itself, and the provider isn't in a position to easily reach across to the AuthorizationDialog. In any case, per your suggestion above the AuthorizationDialog is using the global variable now and so there shouldn't be anything that needs to be reset in the callback.